### PR TITLE
integration: update the gitlab config mandatory fields to match reality

### PR DESCRIPTION
**Pull request by freben**
As a response to https://discord.com/channels/687207715902193673/687207715902193679/808192034359345153.

This one is a bit tricky in terms of deprecation notice time. The corresponding change could be done for at least bitbucket too.

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4420-tf3f0ytmjaudbs6rzkdxciw43tak6j8o.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60210b14353bbb0572a27acf

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4420-tf3f0ytmjaudbs6rzkdxciw43tak6j8o.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60210b14353bbb0572a27acf

File: packages/integration/src/helpers.ts [hunk 1]
```diff
@@ -16,11 +16,22 @@
...
+  check.host = host;
+  return check.host === host;
+}
+
+/** Checks whether the given argument is a valid URL */
+export function isValidUrl(url: string): boolean {
+  try {
+    // eslint-disable-next-line no-new
+    new URL(url);
+    return true;
```
Comment (Rugvip) [inline packages/integration/src/helpers.ts:31]: `return Boolean(new URL(url))` 😎 

... naht

**Review by Rugvip — APPROVED** [review]
Yep, makes sense 👍

**Pull request merged**